database/migrations/2016_07_14_154357_MysqlToPostgres.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Symfony\Component\Console\Output\ConsoleOutput;
+
+class MysqlToPostgres extends Migration
+{
+    private $console;
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $this->console = new ConsoleOutput();
+
+        // Generate pgloader config
+        $mysqlConnection = "from mysql://" . env('DB_USERNAME') . ":" . env('DB_PASSWORD') . "@" . env('DB_HOST') . "/" . env('DB_DATABASE');
+        $postgresConnection = "into postgresql://" . env('POSTGRESQL_DB_USERNAME', 'homestead') . ":" . env('POSTGRESQL_DB_PASSWORD', 'secret') . "@" . env('POSTGRESQL_DB_HOST', 'localhost') . "/" . env('POSTGRESQL_DB_DATABASE', 'homestead');
+
+        $header = "LOAD DATABASE";
+        $body = <<<'EOD'
+with truncate
+
+CAST type datetime to timestamp drop default drop not null using zero-dates-to-null,
+      type date drop not null drop default using zero-dates-to-null
+
+EXCLUDING TABLE NAMES MATCHING 'migrations';
+EOD;
+
+        $output = implode("\n", array($header, $mysqlConnection, $postgresConnection, $body));
+        $configPath = base_path() . "/pfmimport.load";
+        file_put_contents($configPath, $output);
+
+        // Run pgloader
+        $this->execRunWithCallback("pgloader " . $configPath);
```

Comment:
This generated the following message on my dev environment:
`sh: 1: pgloader: not found`

The migration was marked as successful but it looks like `pgloader` needs to be installed or otherwise configured in the Vagrant setup scripts (Ansible as well?).